stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/common/Timestamp.kt

```diff
@@ -0,0 +1,29 @@
+package io.getstream.chat.android.compose.ui.common
+
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import com.getstream.sdk.chat.utils.formatDate
+import io.getstream.chat.android.compose.ui.theme.ChatTheme
+import java.util.Date
+
+/**
+ * Represents a timestamp in the app, that's used primarily for channels and messages.
+ *
+ * @param date - The date to show in the timestamp.
+ * @param modifier - Modifier for styling.
+ * */
+@Composable
+public fun Timestamp(
+    date: Date?,
+    modifier: Modifier = Modifier,
+) {
+    val formatter = ChatTheme.configuration.dateFormatter
+
+    Text(
+        modifier = modifier,
+        text = formatter.formatDate(date),
```

Comment:
We also have a `formatTime` extension that we use in a couple places in the UI Components implementation. Might want to consider having two composables like this, or a parameter that decides how to format.

Reply:
Good point - I was thinking of exposing a formatting function or something, instead of the formatter itself, because that way we can have the same formatter, but override behavior anyway.

Reply:
+1 to Filip point , format function over a class